definitions/customRules.js

```diff
@@ -7,25 +7,21 @@ class OverlaysMerger {
     return 'writeCheck';
   }
 
-  any() {
-    return {
-      onEnter: (node, type, ctx) => {
-        if (node['x-redocly-overlay']) {
-          const definitionDir = path.dirname(ctx.filePath);
-          const overlayPath = path.resolve(definitionDir, node['x-redocly-overlay'].path);
-
-          if (fs.existsSync(overlayPath)) {
-            const patch = JSON.parse(fs.readFileSync(overlayPath));
-
-            Object.keys(patch).forEach((k) => {
-              node[k] = patch[k];
-            });
-
-            delete node['x-redocly-overlay'];
-          }
-        }
-      },
-    };
+  enter(node, type, ctx) {
+    if (node['x-redocly-overlay']) {
```

Comment:
```suggestion
    if (node['x-redoclyOverlay']) {
```

I believe we should use camelCase for extensions. We have one extension that does not which is inconsistent.